src/lib/expert_partitioner/dialogs/format.rb

```diff
@@ -61,27 +57,21 @@ def run
       end
     end
 
-
-  private
+    private
```

Comment:
ok, just expect to fix it if my pr will be merged. Actually this one is already in two yast repositories e.g.
https://github.com/yast/yast-yast2/blob/master/.rubocop.yml#L33
https://github.com/yast/yast-bootloader/blob/master/.rubocop.yml#L95